test/integration/tcp_proxy_integration_test.cc

```diff
@@ -523,25 +523,24 @@ TEST_P(TcpProxyIntegrationTest, TcpProxyNoDataBytesMeter) {
 TEST_P(TcpProxyIntegrationTest, TcpProxyBidirectionalBytesMeter) {
   setupByteMeterAccessLog();
   const int client_count = 3;
-  const int upstream_count = 3;
 
   enableHalfClose(false);
   config_helper_.addConfigModifier([this](envoy::config::bootstrap::v3::Bootstrap& bootstrap) {
```

Comment:
nit: capturing upstream_count in the capture of config helper would be a cleaner solution imo